### PR TITLE
added jwt_token_refresh_interval

### PR DESCRIPTION
**Pull request by kshitijv256**
solves issue [6361](https://github.com/coronasafe/care_fe/issues/6361) in care_fe

### CODE REVIEW

**Review by khavinshankar — APPROVED** [review]

**Review by rithviknishad — CHANGES_REQUESTED** [review]
@kshitijv256 you'll also need to update the [config schema](https://github.com/coronasafe/care_deploy_configs/blob/main/config_schema.yaml)

**Comment by kshitijv256** [comment]
@rithviknishad updated config_schema.yaml

File: config_schema.yaml [hunk 1]
```diff
@@ -21,6 +21,7 @@ required:
   - sample_format_asset_import
   - sample_format_external_result_import
   - enable_hcx
+  - jwt_token_refresh_interval
```
Comment (rithviknishad) [inline config_schema.yaml:24]: This config is not a required config, since we have a fallback set in FE.

```suggestion
```

**Review by rithviknishad — APPROVED** [review]
Nice work!

**Comment by rithviknishad** [comment]
@mathew-alex summary of this change:

`jwt_token_refresh_interval` config can be set to change the JWT token refresh interval. This is not a mandatory config, since the front-end has a fallback defined.

**Pull request merged**